src/components/Help.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import snarkdown from 'snarkdown'
+import Stack from 'cozy-ui/transpiled/react/Stack'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import AvailablePlatforms from './AvailablePlatforms'
+
+// eslint-disable-next-line no-unused-vars
+const DumbHelp = ({ t, f, ...props }) => {
```

Comment:
Nit, but why load the unused vars here?

Reply:
Because since I get all other props and spread them on the underlying element which doesn't take `f` props, I destructured it so it's not part of `props`. But since it's not used, eslint triggers an error.

I spread the props so the component is flexible enough for future use, but it's not required now, so I can remove it if you think it's better ;)

Reply:
Could useI18n be used here ?

Reply:
Yes of course, I forgot about this. Thanks for the reminder!

Reply:
See https://github.com/cozy/cozy-passwords/pull/16/commits/23f589318265a25ff4f96dfce3b5ae0c0767bdb2